### ISSUE TITLE
Data files

### ISSUE THREAD

**Issue by Mattriks** [nickmckay/iso2kNatureGeoscience2023#2]
What data files is it necessary to include in the Github/Zenodo package? In the @georgyfalster code, I see there are `download.file(url)` statements, but the "iso2k1_0_0.RData" file has also been added to this repo. Is the latter necessary, or could you simply give e.g. the doi http://doi.org/10.25921/57j8-vs18  (in a "Data Availability" statement)?

**Comment by Mattriks** [comment]
@nickmckay

**Comment by nickmckay** [comment]
Thanks for the bump - classes started this week so I've been a bit distracted on this. Georgy wrote her code to get all the files from the internet, which is a great idea, and I modified it so that when it downloads them it puts them into a data folder, which then got added to the repository. So in this case it's both. I liked this solution because it's traceable and reproducible, but also convenient to have the data locally. There is a limit to data size we can put on Github though, so this may not work with the model data.

**Comment by Mattriks** [comment]
I see .RData has been added to .gitignore.  You could do the same with other data files e.g. .nc, .mat, etc so they don't get tracked through the commits.

**Comment by nickmckay** [comment]
Right, good idea. Please feel free to add any data types that you're
working with that we shouldn't track and I'll do the same.
